deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,47 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+.opdracht-1  > ul{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  /*height: 300px;*/
+
+  & li {

```

Comment:
Ik ben me niet helemaal zeker waarom dit '&'-teken specifiek naar de list-items uit deze unordered list wijst... ik snap hem wel maar is dat omdat dit de enige lijst in de html is?